client/src/pages/index.jsx

```diff
@@ -1,8 +1,29 @@
 import { SEO } from "@src/components/SEO";
 import AppLayout from "../layout/index.jsx";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { GET_BOARDS } from "../lib/graphql/queries";
+import { useQuery } from "@apollo/client";
+import { useEffect } from "react";
+import { useRouter } from "next/router.js";
 
 export default function Home() {
+	const { loading, error, data } = useQuery(GET_BOARDS);
+	const router = useRouter();
+	useEffect(() => {
+		if (data && data.boards.length > 0) {
+			const firstBoard = data.boards[0];
```

Comment:
Nit and not a blocker so up to you if you want to edit this. But I'd save this in a const. Something like
`const boards = data?.boards`